src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,52 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Class representing Events
+ */
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    protected LocalDate fromDate;
+    protected LocalDate toDate;
+
+    /**
+     * Creates an instance of Event
+     * @param name name of the event
+     * @param from Start date/time of the event
+     * @param to End date/time of the event
+     */
+    public Event(String name, String from, String to) {
+        super(name);
+        this.from = from;
+        this.to = to;
+
+        // Check if they are in dateTime format
+        // If so, update the string
+        try {
+            fromDate = LocalDate.parse(from);
+            this.from = fromDate.format(DateTimeFormatter.ofPattern("MMM d yyyy"));
+
+        } catch (DateTimeParseException e) {}
```

Comment:
Can we try to convey the user about this `DateTimeParseException` instead of just catching it ?